velox/docs/functions/spark/datetime.rst

```diff
@@ -29,13 +29,15 @@ These functions support TIMESTAMP and DATE input types.
     If num_days is a negative value then these amount of days will be
     deducted from start_date.
 
-.. spark:function:: datediff(endDate, startDate) -> integer
+.. spark:function:: date_format(timestamp, format) -> string
 
-    Returns the number of days from startDate to endDate. Only DATE type is allowed
-    for input. ::
+    Converts ``timestamp`` to a date/time string in the format specified by ``format``.
+    The result needs to be adjusted according to local time zone.
```

Comment:
Can we document all the supported patterns, and the behavior for unsupported ones?

Reply:
Updated with commonly used formats listed.